tests/config_authorization/main.tf

```diff
@@ -0,0 +1,27 @@
+provider aws {
+  region = "us-east-1"
+}
+
+module config_authorization {
+  source = "../../"
+
+  authorization = {
+    name       = "tardigrade-config-auth-${random_string.this.result}"
```

Comment:
authorization/variables.tf doesn't have a 'name' field, but I'm assuming you can add to that 'dict' of values and it does no harm?

Reply:
valid, removed. was copy-pasta from the aggregator config.

and yes, you can indeed pass objects with extra "keys" and terraform will just ignore them, but we shouldn't do that for these tests.